pennylane_lightning/core/_serialize.py

```diff
@@ -285,6 +285,9 @@ def _pauli_word(self, observable, wires_map: dict = None):
         def map_wire(wire: int):
             return wires_map[wire] if wires_map else wire
 
+        if len(observable) == 0:
+            return self.named_obs(PAULI_NAME_MAP["I"], [0])
```

Comment:
Would you add a test for this case?

Reply:
It looks strange. It was covered as shown on `codecov` web site [here](https://app.codecov.io/gh/PennyLaneAI/pennylane-lightning/pull/919/blob/pennylane_lightning/core/_serialize.py#L288)  . Do you know why there is a complain here @maliasadi ?

Reply:
Weird! :thinking: -- nwm, might be another false positive case